miqi/runtime/context_runtime.py

```
@@ -376,6 +376,10 @@ def should_auto_compact(
         "claude-haiku-4-5": 200_000,
         "deepseek-chat": 128_000,
         "deepseek-reasoner": 128_000,
+        # v4-flash 实测硬上限 102400（#715 现场记录）；登记后 hard_limit
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)